### DISCUSSION TITLE
Integration of storybook in the documentation

### DISCUSSION THREAD

**Discussion by GiGa1989** [ant-design/ant-design#41148] [Ideas]
It would be very useful to have a storybook page in addition to the [current overview](https://ant.design/components/overview/).

**Comment by zombieJ** [comment]
Could you provide more info about this? What do you want with overview page?

**Reply by GiGa1989** [reply]
Something like [this](https://storybooks-official.netlify.app/?path=/story/addons-knobs-withknobs--tweaks-static-values) with all knobs

**Reply by zombieJ** [reply]
Since I'm not native speaker. What's `all knobs` means?

**Reply by GiGa1989** [reply]
The knobs are used for live interaction with the component.
Practical example with the Option prop of the Tabs component at this [link](https://ant.design/components/tabs#components-tabs-demo-extra) or with the Size prop at this [link](https://ant.design/components/tabs#components-tabs-demo-size).

I hope I was clearer in explaining knobs and why it's important to have a storybook.

**Reply by arvinxx** [reply]
@GiGa1989  dumi will introduce a similar feature , stay tuned for it.
